specification/MDF.tex

```diff
@@ -126,9 +126,13 @@ \subsection{Optional Parameters}
 
 If a group is optional all of its parameters may be omitted, if this group is not used. The groups \inlvar{/}, \inlvar{/study}, \inlvar{/experiment}, \inlvar{/scanner}, \inlvar{/acquisition} contain mostly metadata and are mandatory. The \inlvar{/tracer} group is only mandatory if magnetic material has been placed in the MPI system. The groups \inlvar{/measurement}, \inlvar{/calibration}, and \inlvar{/reconstruction} are all optional. In case of calibration measurements, the \inlvar{/calibration} group is mandatory. The reconstruction data is stored in \inlvar{/reconstruction}. 
 
-\subsection{Parameter Extension}
+\subsection{User defined parameters}
 
-Occasionally, it is necessary to store additional specific parameters or metadata that are not covered in the specifications, such as the temperature of the room in which your MPI device is operated. In this case, you are free to add new parameters to any of the existing groups. Moreover, if necessary, you are also free to introduce new groups. In order to be able to distinguish these datasets and groups from the specified ones, we recommend to use the prefix ``\inlvar{\_}'' for all parameters and groups. As an example, one could add a new group \inlvar{/\_room} that includes the dataset \inlvar{\_temperature}.
+For all user defined parameters or groups the MDF format reserves the prefix `\inlvar{\_}' and the group \inlvar{/detail} in any specified group. User defined parameters/groups within the group \inlvar{/detail} may be stored without the prefix `\inlvar{\_}'. This step is necessary to avoid any name clashes between parameters/groups possibly specified in the future and user defined ones. 
```

Comment:
...and the group \inlvar{/detail} in any specified group.
-> ...and the subgroup \inlvar{.../detail/}